stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+import '../../dist/list.css';
+import '../../dist/icon.css';
+import '../../dist/object-list.css';
+
+export default {
+    title: 'Components/Object List',
+    parameters: {
+        description: `
+Object list is one of the variant for list items.
+It provides a structure for extending standard list with additional content.
+
+##Usage
+**Use the object List if:**
+
+- You want to display a brief details to user.
+
+**Do not use the object List if:**
+
+- You want to display a detailed information to user.
+
+**The object List comprises:**
+ 
+- Introduction text
+- Avatar
+- Object Identfier
+- Object Number
+- Object Attribute
+- Object Marker
+- Object Status
+
+## Structure
+**Object List consists of the following elements along with standard list:**
```

Comment:
I indented to say, in the below structure of object list item classes such as fd-list__item etc are used the same as standard list item do. 
But now I feel it may not be required to mention because it is common to all list variants.